openpype/hosts/fusion/api/menu.py

```diff
@@ -173,8 +173,38 @@ def on_set_framerange_clicked(self):
         set_asset_framerange()
 
 
+def get_qt_app():
+    """Main Qt application."""
+
+    app = QtWidgets.QApplication.instance()
+    if app is None:
+        for attr_name in (
+            "AA_EnableHighDpiScaling",
+            "AA_UseHighDpiPixmaps",
+        ):
+            attr = getattr(QtCore.Qt, attr_name, None)
+            if attr is not None:
+                QtWidgets.QApplication.setAttribute(attr)
+
+        policy = os.getenv("QT_SCALE_FACTOR_ROUNDING_POLICY")
+        if (
+            hasattr(
+                QtWidgets.QApplication, "setHighDpiScaleFactorRoundingPolicy"
+            )
+            and not policy
+        ):
+            QtWidgets.QApplication.setHighDpiScaleFactorRoundingPolicy(
+                QtCore.Qt.HighDpiScaleFactorRoundingPolicy.PassThrough
+            )
+
+        app = QtWidgets.QApplication(sys.argv)
+
+    return app
+
```

Comment:
Shouldn't this just do:
```python
from openpype.tools.utils import get_openpype_qt_app
```

Or how does this differ?

Reply:
Similar to the fix implemented here? https://github.com/ynput/OpenPype/pull/6190

Reply:
`get_openpype_qt_app` should be used only in OpenPype process.

Reply:
Interesting - it was recently also used in Photoshop at least in that linked PR, right?

Anyway, seems a bit verbose to duplicate this logic to Photoshop, Fusion (and likely also needs to be done for After Effects and Resolve, etc. or any host that doesn't has Qt itself)

Reply:
> Interesting - it was recently also used in Photoshop at least in that linked PR, right?

Photoshop and After Effects are using openpype process to show UIs.

Reply:
I still think we're better of performing some more code re-use. But since this is only Fusion now - I guess it's redundant, but if we implement this fix also for e.g. Resolve then exposing the basis of it in `tools.utils` or whatever would be a good way to keep the implementation of it in sync. The `get_openpype_qt_app` could even use the same function to set up the global scaling things, etc.